src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -270,6 +275,28 @@ public async Task RestartHostAsync(CancellationToken cancellationToken)
             }
         }
 
+        private void QueueBackgroundTasks()
+        {
+            var tIgnore = Task.Run(async () =>
+            {
+                // kick off a conditional SyncTriggers operation in the background
+                // the delay is important - it gives the host time to establish
+                // whether it is the primary, and also keeps these operations off
+                // of the cold start path
+                await Task.Delay(TimeSpan.FromSeconds(30));
+                await SyncTriggersAsync();
+            });
```

Comment:
Can we catch/log exceptions coming from this background task? They'll be silently lost otherwise...

Reply:
👍